CONTRIBUTING.md

```diff
@@ -75,7 +75,8 @@ Here is an example of a good one:
 > The algorithm is described at https://wikipedia.org/wiki/McGillicutty_Algorithm
 >
 > Fixes #159
-
+>
```

Comment:
Should the PR type label appear in the commit message? If so, could you provide an example here?

Reply:
I don't think PR type lable should appear in the commit message, it's duplicated.
Contributors just need to set label in the PR website.
<img width="336" alt="image" src="https://github.com/trpc-group/trpc-go/assets/49024930/5fc11b57-7242-4e10-8f97-da7e3ce4f44a">